DigitalLearningSolutions.Web/Controllers/TrackingSystem/CentreConfiguration/RegistrationPromptsController.cs

```diff
@@ -143,6 +143,37 @@ string action
             };
         }
 
+        [HttpGet]
+        [Route("Add/Summary")]
+        [ServiceFilter(typeof(RedirectEmptySessionData<AddRegistrationPromptData>))]
+        public IActionResult AddRegistrationPromptSummary()
+        {
+            var data = TempData.Peek<AddRegistrationPromptData>()!;
+            var promptName = customPromptsService.GetCustomPromptsAlphabeticalList().

```

Comment:
I'm not sure how often this GetCustomPromptsAlphabeticalList is called only to recover a single value - if this occurs multiple times, do you think we should make a different method just to get a specific prompt by Id?

Reply:
This is the only place where that is happening.